worker.py

```diff
@@ -139,17 +139,26 @@
 scene.add(lamp_fill)
 
 
+def translate_quat(pb_quat):
+  """ Convert pyBullet XYZW quaternions into Blender WXYZ quaternions."""
+  x, y, z, w = pb_quat
+  return [w, x, y, z]
+
+
 # --- Dump the simulation data in the renderer
 room = scene.add_from_file(str(floor.vis_filename))
+room.position = floor.position
+room.quaternion = translate_quat(floor.rotation)
```

Comment:
Shouldn't we have ONE standard for quaternions and convert between formats using setter/getters?
I find it is a bit odd the translate_quat is in the user side of the code.

Reply:
agreed. This is a quick fix not a solution.

Reply:
Can we add a TODO and an issue to track it? Thanks